### PR TITLE
property name changed from acceleratedNetworkingEnabled to EnableAcce…

### PR DESCRIPTION
**Pull request by skomarag**
…leratedNetworking

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@skomarag__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (skomar). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by DeepakRajendranMsft** [comment]
1. fix the test
2. increase NRP SDK version
3. once the new a SDK has been published, mail the SDK team to remove the old SDK

**Comment by azurecla** [comment]
Hi __@skomarag__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (skomar). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by skomarag** [comment]
Leave a comment

**Comment by azurecla** [comment]
Hi __@skomarag__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (skomar). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: src/ResourceManagement/DataLake.StoreUploader/DataLakeStoreUploader.Tests/UnitTests/MultipleSegmentUploaderTests.cs [hunk 1]
```diff
@@ -106,7 +106,7 @@ public void MultipleSegmentUploader_MultipleSegments()
         /// <summary>
         /// Tests an uneventful upload from scratch made of several segments
         /// </summary>
-        [Fact]
+        [Fact(Skip ="This test is failing in build machine, toDo: BeGoldSm to fix this")]
```
Comment (begoldsm) [inline src/ResourceManagement/DataLake.StoreUploader/DataLakeStoreUploader.Tests/UnitTests/MultipleSegmentUploaderTests.cs]: I agree and am signed off on this. I will, in general, need to figure out how to properly test our threading logic in a consistent way.

**Comment by begoldsm** [comment]
@skomarag I am signed off on the ADLS test skip change.

**Comment by avijitgupta** [comment]
Looks good.

**Comment by shahabhijeet** [comment]
Even though the property is changed, the Major version was bumped up in the last PR, so this change is inline with the previous change, so no need to bump up the major version again.

**Comment by DeepakRajendranMsft** [comment]
Please make sure to remove the old version from nuget

**Pull request merged**